### PR TITLE
add my works

### PR DESCRIPTION
**Pull request by lgc2333**

### CODE REVIEW

**Comment by Futrime** [comment]
Failed to build, please check again.

**Comment by WangYneos** [comment]
建议不要提交已停更/放弃维护的插件

**Comment by Futrime** [comment]
author field should only contains [a-zA-Z0-9-]+, you are recommended to use your github username.

**Comment by lgc2333** [comment]
okay

File: teeth/formapiex.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "tooth": "github.com/lgc-LLSEDev/FormAPIEx",
+  "information": {
+    "author": "lgc2333",
+    "description": "让处理表单返回值变得更容易！",
+    "homepage": "https://github.com/lgc-LLSEDev/FormAPIEx",
+    "license": "Apache-2.0",
+    "name": "FormAPIEx",
+    "repository": "github.com/lgc-LLSEDev/FormAPIEx",
+    "tags": [
+      "module",
```
Comment (Futrime) [inline teeth/formapiex.json:12]: This is a plugin rather than a module. Module is restricted to ll modules.

File: teeth/navigationapi.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "format_version": 1,
+  "tooth": "github.com/lgc-LLSEDev/NavigationAPI",
+  "information": {
+    "author": "lgc2333",
+    "description": "其他插件可对接的导航 API",
+    "homepage": "https://github.com/lgc-LLSEDev/NavigationAPI",
+    "license": "Apache-2.0",
+    "name": "NavigationAPI",
+    "repository": "github.com/lgc-LLSEDev/NavigationAPI",
+    "tags": ["module", "llse", "navigate", "navigation", "api", "navigationapi"]
```
Comment (Futrime) [inline teeth/navigationapi.json:11]: Not a module

**Review by Futrime — CHANGES_REQUESTED** [review]

**Review by Futrime — APPROVED** [review]

**Pull request merged**